deploy/config/openshift/test-env.sh.yml

```diff
@@ -30,7 +30,8 @@ spec:
     spec:
       serviceAccountName: ${APP_NAMESPACE_NAME}-sa
       containers:
-      - image: debian
+      - image: '${PULL_DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}/secrets-provider:latest'
```

Comment:
It may not really matter because this is just a test, dummy container but was thinking that because these are testing, should we be using the one we built as part of the run `secrets-provider-for-k8s:latest` (see `build`)?
Also this image appears in k8s/test-env.sh.yml and also secrets-provider-init-container.sh.yml so which ever you choose, shouldn't we also be percing the change to them as well?

Reply:
My understanding is that this is purely just a test container. It only needs to satisfy the following:
1. to have `sleep` so it can initially sleep
2. to have `cut`, `grep` and `printenv` for making test assertions
Debian was chosen because it meets these requirements. Incidentally the secrets-provider container image meets these requirements too.

I opted to use the secrets-provider container image because it is already being pushed to the OpenShift registry as part of the pipeline. The alternative is to push debian, which honestly might be the more honest approach.

> Also this image appears in k8s/test-env.sh.yml and also secrets-provider-init-container.sh.yml 
The reason why those are being left out is because the dockerhub pull limits don't seem to be an issue in k8s, only in OpenShift.

Reply:
Another option might be to use `cyberark/conjur-cli:5-latest` for the test container, since that will be getting pulled already when `cyberark/kubernetes-conjur-deploy` scripts are cloned and run, and `conjur-cli` has the `cut`, `grep`, and `printenv`.  It might be more palatable using `conjur-cli` to test rather than secretless-provider?

Or, if we get a team DockerHub account for a bigger download rate? It may come to that eventually.

Reply:
@diverdane

It's true that `cyberark/conjur-cli:5-latest` is already available but the issue there is that we have to inspect the internals of `cyberark/kubernetes-conjur-deploy` to determine what the image name is, and in doing so we couple the 2 repos together. If that image name changes in the deploy scripts then that impacts this one.

Infra suggest pushing images into the repository as the go to vs using a Dockerhub account.

To be honest, pushing `debian` isn't all that involved. I think I'll just go in that direction